cypress/integration/main-view-spec.js

```diff
@@ -56,4 +56,13 @@ describe('the main view', () => {
       .select('Denver')
       .should('have.value', 'Denver')
   });
+
+  it('should be able to move to the faq page', () => { 
+    cy
+      .get('a')
+      .click()
+
+      .get('img')
+      .should('have.attr', 'src', '/static/media/AQI-chart.7b1ed9f0.jpg')
+  });
```

Comment:
Can we add more test for all the elements that appear on the FAQ page? I think we should test as much as possible.

Reply:
Yep. I'm putting those on a different branch that continues off of this branch!

Reply:
awesome! Thank you.